tests/phpunit/CRM/Export/BAO/ExportTest.php

```diff
@@ -398,21 +400,22 @@ public function setUpMembershipExportData() {
     $this->setUpContactExportData();
     // Create an extra so we don't get false passes due to 1
     $this->contactMembershipCreate(['contact_id' => $this->contactIDs[0]]);
```

Comment:
switch to using a set up function that creates autorenew & is also the one used in the test below which DOES return a result for auto-renew - but through a mechanism not available here